studio/stores/authConfig/schema/AuthProviders/AuthProvidersFormValidation.tsx

```diff
@@ -884,7 +884,7 @@ const EXTERNAL_PROVIDER_GOOGLE = {
     iconKey: 'google-icon',
     requiresRedirect: true,
     helper: `Register this callback URL when using Sign-in with Google on the web using OAuth.
-            [Learn more](https://supabase.com/docs/guides/auth/social-login/auth-apple#configure-your-services-id)`,
+            [Learn more](https://supabase.com/docs/guides/auth/social-login/auth-google#configure-your-services-id)`,
```

Comment:
Wondering if the slug should be updated here. Maybe https://supabase.com/docs/guides/auth/social-login/auth-apple#configuration-web?